docs/src/use/configure/migration-guide.md

```diff
@@ -0,0 +1,422 @@
+---
+title: Configuration Migration Guide
+eleventyNavigation:
+    key: migration guide
+    parent: configure
+    title: Configuration Migration Guide
+    order: 8
+---
+
+This guide provides an overview of how you can migrate your ESLint configuration file from the `.eslintrc` format to the new configuration file format, `eslint.config.js`.
```

Comment:
Because these config formats can exist outside of files, I think it's helpful to name them and then refer to the names throughout rather than the filenames.

```suggestion
This guide provides an overview of how you can migrate your ESLint configuration file from the eslintrc format (typically configured in `.eslintrc.js` or `.eslintrc.json` files) to the new flat config format (typically configured in an `eslint.config.js` file).
```

Reply:
will update throughout as well